### PR TITLE
Allow keyspace to be null

### PR DESCRIPTION
**Pull request by NastasiaSabyZenika**
Separation between client configuration and connection configuration (with keyspaces defined).
We do not have to define a keyspace for a new client.

### CODE REVIEW

File: src/Tests/Units/DependencyInjection/M6WebCassandraExtension.php [hunk 1]
```diff
@@ -205,21 +241,27 @@ public function testMulticlientsConfig()
...
      * @dataProvider unexpectedConfigValueDataProvider
      */
     public function testUnexpectedValueConfig($configs)
     {
         $parameterBag = new ParameterBag(array('kernel.debug' => true));
         $container = new ContainerBuilder($parameterBag);
 
-        $this->if($extension = new TestedClass())
-            ->exception(function() use($extension, $configs, $container) {
+        $this->if ($extension = new TestedClass())
```
Comment (Oliboy50) [inline src/Tests/Units/DependencyInjection/M6WebCassandraExtension.php]: `if ()` => it's weird to have a space between a methodName and its bracket

File: src/Tests/Units/DependencyInjection/M6WebCassandraExtension.php [hunk 2]
```diff
@@ -236,16 +278,20 @@ public function testInvalidConfig()
...
```
Comment (Oliboy50) [inline src/Tests/Units/DependencyInjection/M6WebCassandraExtension.php]: same

**Review by Oliboy50 — APPROVED** [review]

File: src/Tests/Units/DependencyInjection/M6WebCassandraExtension.php (hunk 1 above)
Comment (t-geindre) [reply src/Tests/Units/DependencyInjection/M6WebCassandraExtension.php]: bracket ? `{` :trollface:

Comment (Oliboy50) [reply src/Tests/Units/DependencyInjection/M6WebCassandraExtension.php]: [ ] => Brackets
( ) => Round brackets or parentheses
{ } => Curly brackets or braces
〈 〉=> Angle brackets or chevrons

`bracket` works for everything :trollface:

**Review by gmillet — APPROVED** [review]

**Review by lnahiro — APPROVED** [review]

File: src/Tests/Units/DependencyInjection/M6WebCassandraExtension.php [hunk 3]
```diff
@@ -213,13 +252,16 @@ public function testUnexpectedValueConfig($configs)
         $container = new ContainerBuilder($parameterBag);
 
         $this->if($extension = new TestedClass())
-            ->exception(function() use($extension, $configs, $container) {
+            ->exception(function () use ($extension, $configs, $container) {
                 $extension->load($configs, $container);
             })
             ->isInstanceOf('\Symfony\Component\Config\Definition\Exception\InvalidConfigurationException');
```
Comment (gflament) [inline src/Tests/Units/DependencyInjection/M6WebCassandraExtension.php]: Petite ligne en trop ?

**Review by gflament — APPROVED** [review]

**Pull request merged**